dockerclient.go

```diff
@@ -306,6 +306,33 @@ func (client *DockerClient) getStats(id string, cb StatCallback, ec chan error,
 	}
 }
 
+func (client *DockerClient) GetStats(id string, statsChan chan Stats, errorChan chan ContainerError, exitChan chan bool) {
+	uri := fmt.Sprintf("%s/%s/containers/%s/stats", client.URL.String(), APIVersion, id)
+	resp, err := client.HTTPClient.Get(uri)
+	if err != nil {
+		errorChan <- ContainerError{Error: err, ContainerId: id}
```

Comment:
I agree with the errorChan when the stats stream is open, but should we use an error code return for opening the stream (and check this http req)?

I am not sure about this one (what's the easiest for the dev), it's an honest question. Also, tests?